components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -145,6 +145,7 @@ public class SCIMUserManager implements UserManager {
     private static final String SQL_FILTERING_DELIMITER = "%";
     private static final String ERROR_CODE_INVALID_USERNAME = "31301";
     private static final String ERROR_CODE_INVALID_CREDENTIAL = "30003";
+    private static final String ERROR_CODE_NON_EXISTING_USER = "30007";
```

Comment:
Can't we use the defined constant for this error in the user.core?

Reply:
Fixed